scalafix-core/src/main/scala/scalafix/syntax/package.scala

```diff
@@ -70,7 +70,14 @@ package object syntax {
   }
 
   implicit class XtensionToken(token: Token) {
-    def posTuple: (Int, Int) = token.start -> token.end
+    // copy pasted from:
+    // github.com/scalameta/scalafmt/blob/master/scalafmt-core/shared/src/main/scala/org/scalafmt/util/TokenOps.scala
```

Comment:
What do you mean?
scalameta/scalafmt/scalafmt-core/shared/src/main/scala/org/scalafmt/util/TokenOps.scala
Like this?